src/components/card/CardMedia/CardMedia.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import classnames from 'classnames'
+
+interface CardMediaProps {
+  exdent?: boolean
```

Comment:
also a nitpick from testing, it looks like `inset` is another option for CardMedia that we might want to add as a prop (adds class `usa-card__media--inset`, see https://components.designsystem.digital.gov/components/detail/card--test.html)

Reply:
good catch